### DISCUSSION TITLE
Welcome to clarion Discussions!

### DISCUSSION THREAD

**Discussion by bytemaster** [bytemaster/clarion#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by a1zanini** [comment]
How how does the governance of this project works?

Is the code open-source?

**Reply by bytemaster** [reply]
Everything is going to be open source. There is no centralized governance or token-based voting. 

Eventually I hope to integrate governance ideas from my book, moreequalanimals.com as a means of governing.

**Reply by a1zanini** [reply]
Thanks for the reply Dan. 

I'm not a developer but I was involved in the creation of a taxi hailing app with more than 10 million downloads. 

One thing I've noticed in the crypto space is the lack of focus in UX and UI of some projects. This is key for massive adoption. When you see binance, you can tell they have an interdisciplinary team which is constantly understanding the needs from their users. 

I can help in the design process. Hope this is the right channel

**Reply by bytemaster** [reply]
That will definitely be a focus. We need excellent ui or it's all for nothing.

**Reply by telos-justin** [reply]
@a1zanini is Binance a positive example of UX? Cant say i'm a fan of their UX but i could be an outlier :-)

**Reply by iamveritas** [reply]
well they are one of the most advanced ones in the field. of course they could improve.
also it seems they are constantly working on improving their UX.

**Comment by mike623317** [comment]
This project sounds wonderful. I really am excited to see this happen. I really do. hope you can tie this in with EOS as much as possible because this type of project could be very popular and help build a positive community - something that EOS desperately needs. A positive community spirit and some tokens on EOS please.

**Comment by abhi3700** [comment]
I personally don't like the governance of EOS in terms of "one token, one vote"...it means that rich people are going to dominate the network.

May be in this project, you can think of bettering the Governance, which is another essential pillar of any dApp, Blockchain.

**Reply by bytemaster** [reply]
This project is not DPOS.  I will never do another DPOS chain.

**Reply by McCarthyJohn** [reply]
Never is a very long time

**Reply by chadrickm** [reply]
> I will never do another DPOS chain.

Does this mean that the existing DPOS chain(s) (EOS) are good enough that you will never need another or that you are souring on DPOS at the moment? If so, why?

**Reply by abhi3700** [reply]
> This project is not DPOS. I will never do another DPOS chain.

Are u going to build like Steemit (a Blockchain + dApp) or a new Blockchain with new Consensus algorithm?

**Reply by iamveritas** [reply]
> This project is not DPOS. I will never do another DPOS chain.

You are the inventor of dpos though. Can you please elaborate on this decision?

**Reply by MarkStair** [reply]
> This project is not DPOS. I will never do another DPOS chain.

Can you elaborate on why you would never do another DPOS chain?  I guess this means using eosio as a platform is out of the question or would their be a way to adapt eosio into POS or other approach?

**Reply by ghost** [reply]
I guess he had a contract with b1 when he left, thus EOSIO will not have another competitor. That does not mean he has abandoned DPOS.

**Comment by cifilter** [comment]
How will messages be validated as authentic? Will a traditional user/password account hash a private key that signs all messages/data as authoritatively originating from that user?

**Reply by Tomsocode** [reply]
I'm really excited, looking forward to what will come out of this

**Reply by bytemaster** [reply]
Yes. Everything will be signed by private keys as the root of identity.

**Comment by wenre** [comment]
When can I use it?

**Reply by bytemaster** [reply]
The project is just getting started. I hope for a mvp in 6 months. No promises.

**Reply by bewithram** [reply]
Projects like Holochain and Status are still struggling to deliver on similar vision after working for years. Do you think this 6 months MVP is realistic ?

**Comment by tomcode1000** [comment]
This is a nice idea

**Comment by UrsaPolarisRecords** [comment]
It would be nice to be able to send streams of data directly to people. Also, to have “one to one asynchronous” communications include “One to exclusive-many” communications, like a group text message.

I’m curious, where does the data “live” in the Clarion network, before it has been transmitted to the receiving user’s device? Is it hosted locally on the sender’s phone, and accessed via IPFS? That might open up security concerns, which could possibly be avoided by utilizing an accounts system similar to EOSio to verify receiver identity and a multisig threshold to allow the sender to confirm allowing access to the data. 

The specific use case I’m imagining this around is something I’ve tried to do with current tools, with little success. I’d like to be able to send a playlist of songwriting voice memos to a group of people who can then access and add to it. It’s clunky to do with Dropbox and google drive, doesn’t work right in mms or iMessage, email always seems to get lost in the fray, and any other solution requires that you download a new app. Just a thought from someone who can imagine a use case in which sending arbitrarily large messages directly and securely to others might be handy. 

Perhaps the use case of a potential token on this network would be as a storage layer for arbitrarily large files. Run an EOSio blockchain dedicated to NET storage, and don’t prune the chain so people can access IPFS files from long ago. Run Clarion as level 0 for small data, which then interfaces with the blockchain to purchase IPFS storage, and includes the IPFS hash in the level 0 transaction. 

I’m still confused as to the exact mechanism of communication from one device to another. Is it using the regular internet? WiFi meshes? Bluetooth? All of the above? How do we make it so that there is no single point of failure?

Thanks for sharing your thoughts around these things. We need to create the world we want to live in, and create systems so much better than our current governance systems that they have no choice but to fade into obscurity. You seem to be one of the people who are most committed to this goal.

**Reply by bytemaster** [reply]
Thanks for your thoughts. You're node would upload it to several different friends who happen to be online. They could pin it to ipfs. 

I will write another discussion with more details. This is just getting started.

**Comment by Dexaran** [comment]
>To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Greetings Clarion community! 

I'm an anonymous developer and security engineer.

- Fought the flaw of the most popular Ethereum token standard since 2017. ([ERC223](https://github.com/ethereum/eips/issues/223))
- [DDOSed EOS mainnet in 2019](https://www.eosgo.io/blog/Eos-network-congestion-by-ddos-analysis).
- Designed [51%-attack solutions](https://twitter.com/Dexaran/status/1295470808257748995) for POW chains.
- Attempted to create a trustless [on-chain messaging system](https://github.com/EthereumCommonwealth/Roadmap/issues/38) and [fight centralized control of media resources of community-driven projects](https://github.com/ethereumproject/ECIPs/issues/93) at Ethereum (and ETC).

I am interested in this project because I need tools that are not currently available in the world:

- Trustless communication tool
- Decentralized application development / distribution platform that enables the creation of serverless applications (at least at the level of complexity of a simple computer game)
- A documentation control and management tool for decentralized and open projects

I would be happy to be a part of the Clarion community.

Feel free to ask any questions related to smart-contracts / security: dexaran@ethereumclassic.org / dexaran@callisto.network

**Reply by bytemaster** [reply]
Welcome aboard!

**Reply by noypi** [reply]
You need a project like new kind network, i dont know if they still live today

**Comment by jbuck84** [comment]
I'd like to be able to whitelist User X such that I can see posts from anyone they follow. Adds a bit of 'reputation' to the ecosystem to help scale the distance any one post may travel. 

If there is a working group around topics like this, and governance in general, I would love to join the effort. Also very happy to test backend/frontend functionality and offer feedback.

**Reply by bytemaster** [reply]
Fantastic concept!   I created an issue to track this idea.

**Comment by senevrensel** [comment]
Hello everyone !

May be it may sound a bit weird but what  about implementing the project to the POW for a short period and then stop mining after a while . Old school style !
We really do miss POW projects . - But projects of devs' who have reputation in the crypto eco-system.-
Decentralized social media really excites me so much like the idea of Bitcoin. Privacy in communication is the key figure nowadays and will be much more important for the coming years. Wow ! while i am writing really excites me . Billions of people will be inside of that. 
And what i believe is ; if this project will be for communication how about mining thru smart-phones at least for a short period of time so that you can inform and get the attention of people much more . Not only the poople in crypto but also the poople out of crypto. There are millions of people out there. 
Cos' social media means "smart phones" nothing else imo. Other social communication channels mostly used by smart phones.
and also if you can do smth like that ;  people who has zero knowledge about POW mining can be integrated into the first steps of project instead of buying the token. 
Wish you the best for this one. 
I am not a dev nor a computer engineer but if any help needed i am here at least to help the project to advertise for my country. 

Best

**Comment by maxmuze** [comment]
@bytemaster Professionally I’m in the QA space and I’ve been wanting to get into BlockChain QA... the project looks very interesting and I’d be happy to contribute, learn and help where I can..

**Reply by bytemaster** [reply]
I appreciate your interest. You should know that this isn't a Blockchain project (though it could add a blockchain integration in the future).  As we start to get more mature and into testing/QA your contributions will be very valuable.

**Comment by tradingartofwar** [comment]
Enjoying your book (More Equal Animals), Daniel.

**Comment by ma-da** [comment]
I'm not a dev but I like this project and will contribute however I can.

**Comment by sylvaincormier** [comment]
Hi I have read your book and I totally dig your views.
I wrote a paper before I got into blockchain(my first experience was EOS) about a concept called "Augmented democracy".
Although highly idealistic, this concept seeks to make this world a more just and equitable one.
https://github.com/sylvaincormier/augmented-democracy/blob/master/doc/augmented-democracy.pdf
I have joined the latest hackathon hoping to gamify this so this boring concept could somehow become more interesting.
Augmented democracy will especially satisfy #4 and #5 of your definition of a true democracy(page 47). I have also decided to incorporate concepts from the KILT protocol into this and would love to incorporate your concept of dividing up the votes into more local entities(Did I understand that right?) I need help to do this or would gladly join any effort in this direction if this is the direction that Clarion is taking.

**Reply by tradingartofwar** [reply]
My son. Lucius, who is 12 loves programming and is pretty fast on the keyboard.  Any tips on how he could help?

**Reply by sylvaincormier** [reply]
Did you look at the repository? There are images in there explaining the gaming aspect. Funny fact, I have a 13 y.o. already contributing to the project. Sure, please contact me: sylvaincormier@protonmail.com with more info about which programming language he is using and I will try to find something.

**Reply by bytemaster** [reply]
It is always nice to meet someone who appreciates the ideas in my book. I am also working on gamification of the concepts in More Equal Animals.  Eventually this gamification can become an onboarding strategy for Clarion

**Reply by tradingartofwar** [reply]
Thank you for your thoughts.  I'll look around a bit.

**Comment by cmadh** [comment]
We are actively looking for ways to integrate chat into an EOSIO-based game without having to rely on centralized software/servers. 

I wonder if we could integrate Clarion and how much effort it would take to implement a Unity3D-library/package that each community could adopt in other games. 

We are developing in Unity3D, I think integrating it into WebGL would be a smaller problem, communication between game frontend and Clarion-wasm could be done via jslibs - but I wonder how we could make it native, so that Clarion would also be available natively on various desktop-OS and mobile.

**Comment by kanedaron** [comment]
Hello Clarion,

I got interested in blockchain recently, and i wish that someone make a completely decentralized social network which would enable people to freely exchange ideas without a central authority, in my opinion its an urgent matter.

I'm not a developer with much experience but if there's need, i can help with code, copywriting, design or anything which might advance the project even a little.

**Comment by abhi3700** [comment]
How to ensure unique voting w/o KYC in a public permissionless Blockchains?

Voting is a key component of any Governance.
